pyiron_atomistics/sphinx/parser_base.py

```diff
@@ -0,0 +1,188 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+__author__ = "Christoph Freysoldt"
+__copyright__ = (
+    "Copyright 2023, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Christoph Freysoldt"
+__email__ = "freysoldt@mpie.de"
+__status__ = "production"
+__date__ = "Dec 8, 2023"
+
+import re
+from types import GeneratorType
+import numpy
```

Comment:
You don't want to use `np` instead?

Reply:
> You don't want to use `np` instead?

I currently use numpy in a single place for a single function (in get_vector), and I felt using the fully qualified name (numpy.fromstring) improves readability. But I am flexible if you prefer np over numpy.